install.sh

```diff
@@ -71,6 +71,45 @@ if (($SUPPORTS_SSE42 == 0)); then
     exit 1
 fi
 
+replace_tsdb() {
+    if (
+        [ -f "$SENTRY_CONFIG_PY" ] &&
+        ! grep -xq 'SENTRY_TSDB = "sentry.tsdb.redissnuba.RedisSnubaTSDB"' "$SENTRY_CONFIG_PY"
+    ); then
+        tsdb_settings="SENTRY_TSDB = \"sentry.tsdb.redissnuba.RedisSnubaTSDB\"
+
+# Automatic switchover 90 days after $(date). Can be removed afterwards.
+SENTRY_TSDB_OPTIONS = {\"switchover_timestamp\": $(date +%s) + (90 * 24 * 3600)}"
+
+        if grep -q 'SENTRY_TSDB_OPTIONS = ' "$SENTRY_CONFIG_PY"; then
+            echo "Not attempting automatic TSDB migration due to presence of SENTRY_TSDB_OPTIONS"
```

Comment:
I'd say we should limit this case to only when `switchover_timestamp` is defined. The mere existence of `SENTRY_TSDB_OPTIONS` is to large of a blanket, right?

Also we should probably point people to some useful page explaining what is happening. The link can be this PR itself if it has a meaningful body.

Reply:
Ping?

Reply:
Sorry, forgot to respond. I would argue that in the case of `SENTRY_TSDB_OPTIONS` we cannot safely rewrite the config because it *might* be multiline.

Reply:
I think we can do something like this: https://github.com/getsentry/onpremise/pull/470/files#diff-3fbb47e318cd8802bd325e7da9aaabe8R198

That said I'll merge this as it is now and see if we need it later.